spec/features/search_filters_spec.rb

```diff
@@ -187,16 +187,18 @@
     visit root_path
     within(".blacklight-subject_topic_facet") do
       expect(page).not_to have_selector(".card-block", :visible => true)
-      click_link "Topic"
+      find(".card-header").click
```

Comment:
How is Solr 7.2 affecting the front end here?

Reply:
No idea, but I'm getting test failures on the branch without it. Testing in the browser works fine.